widget/select.go

```diff
@@ -195,6 +195,19 @@ func (s *Select) CreateRenderer() fyne.WidgetRenderer {
 	return &selectRenderer{icon, text, shadow, objects, s}
 }
 
+// ClearSelected clears the current option of the select widget.  After
+// clearing the current option, the Select widgets PlaceHolder will
+// be displayed.
+func (s *Select) ClearSelected() {
```

Comment:
The functionality of `ClearSelected()` should be supported by `SetSelected()` by always accepting the empty string. `ClearSelected()` could either be removed or kept as a convenience call for `SetSelected("")` then.
I'm undecided if the `ClearSelected()` should stay then. @andydotxyz what do you think?

The consolidation into `SetSelected()` would remove the double implementation of the `OnChanged` logic.

Reply:
Great feedback.  I wasn't sure which path would be better here.